styles/styles.css

```
@@ -0,0 +1,447 @@
+
+}
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+body {
+    font-family: Roboto, Arial, Helvetica, sans-serif;
+    font-weight: 200;
+    font-size: 20px;
+}
+
+h1, h2, h3, h4, h5, h6 {
+    font-family: Merriweather, "Times New Roman", serif;
+
+}
+h3 {
+    font-weight: 300;
+    font-size: 35px;
+}
+.text-link-vp {
+    color: var(--primary);
+    font-weight: 400;
+    font-size: 16px;
+    text-transform: uppercase;
+}
+.blok-text {
+    font-size: 18px;
+}
+.footer-text_1, .footer-text_2 {
+    color: whitesmoke;
+}
+.footer-text_1 {
+    font-family: Merriweather, "Times New Roman", serif;
+    font-size: 50px;
+}
+.footer-text_2 {
+    font-size: 30px;
+}
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+/* --------------- 3.1 Reusables ----------------------- */
+.outer-content-container {
+    /*border: greenyellow 2px solid;*/
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    flex-shrink: 1;
+    flex-grow: 0;
+    position: relative;
+    background: rgb(72,69,228);
+    background: linear-gradient(63deg, rgba(72,69,228,1) 28%, rgba(87,34,205,1) 88%);
+}
+.inner-content-container {
+    /*border: red 2px solid;*/
+    display: flex;
+    flex-direction: column;
+    align-self: center;
+    text-align: center;
+    padding-top: 50px;
+    padding-bottom: 200px;
+    width: 1050px;
+    column-gap: 0;
+    /*position: relative;*/
+    flex-grow: 0;
+    flex-shrink: 1;
+}
+.inner-content__text {
+    /*border: #ED4385 2px solid;*/
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 22px 0;
+    flex-shrink: 1;
+    flex-grow: 0;
+
+}
+
+button {
+    background-color: var(--secondary);
+    padding: 23px 32px;
+    color: whitesmoke;
+    border: transparent;
+    margin-top: 30px;
+    margin-left: -5px;
+    border-radius: 50px;
+    outline: transparent;
+    width: 340px;
+    height: 80px;
+    align-self: center;
+    font-size: 20px;
+    text-transform: uppercase;
+    box-shadow: 0 7px 4px rgba(0, 0, 0, 0.26);
+
+}
+/* --------------- 3.2 Other --------------------------- */
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
+.outer-main-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    flex-shrink: 1;
+    flex-grow: 0;
+    padding-top: 150px;
+    /*position: relative;*/
+    background-color: var(--grey);
+
+}
+.inner-main-container {
+    /*border: #ED4385 1px solid;*/
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-self: center;
+    text-align: left;
+    padding-top: 80px;
+    padding-bottom: 100px;
+    width: 85%;
+    column-gap: 0;
+    /*position: absolute;*/
+    flex-grow: 0;
+    flex-shrink: 1;
+}
```

Ik zou hier nog
```suggestion
    align-items: center;
}
```
aan toevoegen, zodat de afbeelding gecentreerd staat met de tekst